packages/shorebird_cli/lib/src/commands/patch/patch_command.dart

```diff
@@ -30,6 +30,14 @@ class PatchCommand extends ShorebirdCommand {
   }) {
     _resolvePatcher = resolvePatcher ?? getPatcher;
     argParser
+      ..addMultiOption(
+        CommonArguments.dartDefineArg.name,
+        help: CommonArguments.dartDefineArg.description,
+      )
```

Comment:
This just begs for us to add a addMultiOptionArg(CommonArguments.dartDefineArg) flavor 🤣

Reply:
Yeah, I think that'd be a nice extension of this `CommandArguments` class.